.github/workflows/prerelease.yml

```diff
@@ -254,6 +254,8 @@ jobs:
     - name: Get token
       run: scripts/decrypt_gha_secret.sh scripts/gha-encrypted/prerelease-testing-token.txt.gpg \
           bot-access.txt "$bot_token_secret"
+    - name: Xcode
+      run: sudo xcode-select -s /Applications/Xcode_16.app/Contents/Developer
```

Comment:
Ok! Will do. Do you think we are losing important testing coverage by switching this to Xcode 16? I'm thinking it may be fine since we still lint with Xcode 15.

Reply:
Agreed.